### DISCUSSION TITLE
Protractor/Jasmine to WebdriverIO Mocha migration

### DISCUSSION THREAD

**Discussion by Bajibabup** [webdriverio/codemod#84] [General]
Hi,
We have an existing Protractor Framework with Jasmine/Java Script. Now, we have a plan to migrate to WebdriverIO.
Is it wise to setup a new WebdriverIO project with Mocha(s it has best features) and migrate the existing Protractor/Jasmine scripts? Codemode and other migration features supports it? 
or, is it wise to setup a new project also with Jasmine?
Any inputs are appreciated on it.

**Comment by christian-bromann** [comment]
Yeah totally. The codemod will help you to transform Protractor code into WebdriverIO. Note that this is likely not a 100% full automated process and manual adjustments are required. Check out https://webdriver.io/docs/protractor-migration for more information. Let me know if you have any further questions.

**Reply by Bajibabup** [reply]
Thanks for your response and confirmation!!
